app/controllers/api/physical_servers_controller.rb

```diff
@@ -40,6 +40,16 @@ def restart_mgmt_controller_resource(type, id, _data)
       change_resource_state(:restart_mgmt_controller, type, id)
     end
 
+    def refresh_resource(type, id = nil, _data = nil)
```

Comment:
Is there a reason for the default `id = nil` here? Does anything call this without the second argument?

Reply:
You are right. No need for this here.